Command/DeleteUserCommand.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\Command;
+
+use SumoCoders\FrameworkMultiUserBundle\User\UserRepository;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class DeleteUserCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('sumocoders:multiuser:delete')
+            ->setDescription('Delete a user entity')
+            ->addArgument(
+                'username',
+                InputArgument::REQUIRED,
+                'The username of the user'
+            )
+            ->addOption(
+                'class',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'The class off the user',
+                'SumoCoders\FrameworkMultiUserBundle\User\User'
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $userClass = $input->getOption('class');
+
+        $repository = $this->getRepository($userClass);
+
+        $handler = new DeleteUserHandler($repository);
```

Comment:
ik zou de handler pas aanmaken na dat je het command maakt. Er is nog validatie hierna.